src/components/stepper-item/stepper-item.tsx

```diff
@@ -53,12 +53,26 @@ export class StepperItem implements InteractiveComponent {
   /** is the step disabled and not navigable to by a user */
   @Prop({ reflect: true }) disabled = false;
 
-  /** pass a title for the stepper item */
+  /**
+   * pass a title for the stepper item
+   *
+   * @deprecated use heading instead
+   */
   @Prop() itemTitle?: string;
 
-  /** pass a title for the stepper item */
+  /** stepper item heading */
+  @Prop() heading?: string;
+
+  /**
+   * pass a title for the stepper item
+   *
+   * @deprecated use description instead
```

Comment:
For consistency, could we change the `@deprecated` line to?:
**Use "description" for the component description text.**